addon/templates/components/frost-action-bar.hbs

```diff
@@ -13,13 +13,15 @@
 </div>
 <div class='{{css}}-buttons' data-test={{hook 'frost-action-bar-buttons'}}>
   {{#if _moreActions.length}}
-    {{#frost-button
+    <div 
       class='frost-more-button'
-      hook='moreActions'
-      priority='secondary'
-      size='medium'
-      text=moreActionsText
-    }}
+      data-test={{hook 'moreActions'}}>
+      {{frost-button
+        hook='frost-more-button-button'
```

Comment:
`frost-more-button`?

Reply:
That is ugly naming, but the wrapper that has all the styling is called frost-more-button.

Reply:
lol

Reply:
Would it make since to call this `moreActions-button` since the parent has a hook of `moreActions`?